Datagrid/ListMapper.php

```diff
@@ -124,6 +124,34 @@ public function add($name, $type = null, array $fieldDescriptionOptions = array(
             );
         }
 
+        // NEXT_MAJOR: Remove first "if" statement (when requirement of Symfony is >= 2.8)
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) { // symfony >= 2.8
+            $choices = $fieldDescription->getOption('choices');
+            if (($type == 'choice' || $type == 'Symfony\Component\Form\Extension\Core\Type\ChoiceType') && $choices) {
+                // NEXT_MAJOR: Remove, leave only code from "else" (when requirement of Symfony is >= 3.0)
+                if (method_exists('Symfony\Component\Form\FormTypeInterface', 'setDefaultOptions')) { // 2.8 <= symfony < 3.0
+                    $choicesAsValues = $fieldDescription->getOption('choices_as_values') ?: false;
+                    if (true !== $choicesAsValues) {
+                        @trigger_error(sprintf('The value "false" for the "choices_as_values" option for field `%s` in `%s` is deprecated since symfony version 2.8 and will not be supported anymore in symfony 3.0. Set this option to "true" and flip the contents of the "choices" option instead.', $fieldDescription->getName(), get_class($this->admin)), E_USER_DEPRECATED);
```

Comment:
I don't think we have to handle the symfony option deprecation our self, we just have to avoid using it internally.